### ISSUE TITLE
Unable to create codespace despite plan and resource being visible in Azure portal

### ISSUE THREAD

**Issue by danysdragons** [MicrosoftDocs/vsonline#535]
I wanted to test out the browser-based version of Visual Studio Codespaces. I am using my Azure account for michael.hamel@canada.ca

When trying to get set up at https://online.visualstudio.com/environments:	
-	I pressed the “Create Codespace” button, and was prompted to create a billing plan.
-	I selected my Azure subscription “Visual Studio Enterprise” and pressed “Create”.
-	Saw an error message “The resource validation failed”.

I did some ~~Googling~~ (sorry, I meant Binging), and saw a similar-looking issue here, but not sure if it’s truly a duplicate: https://github.com/MicrosoftDocs/vsonline/issues/452

Based on what I read, I went into the Azure portal and removed plans that had been in fact been created in an earlier attempt to get started. I removed all those plans except 
**vso-plan-eastus**, with the resource **vso-rg-d1536b6,** but I’m still not able to create a Codespace.

According to the Azure portal, I have the plan and resource group below:

**vso-plan-eastus
vso-rg-d1536b6**

![Visual Studio Online Plan](https://user-images.githubusercontent.com/40280395/82750820-e0233580-9d80-11ea-90e2-6a7d645d3482.png)

Visual Studio Codespaces is still showing no existing plans in the dropdown, so that the Create Codespace button just takes me to the menu to create a plan again.

![You Dont have codespaces](https://user-images.githubusercontent.com/40280395/82750843-11036a80-9d81-11ea-867f-21962236005b.png)

Any suggestions?

**Comment by jkeech** [comment]
> * Saw an error message “The resource validation failed”.

We need to improve the error message, but this usually means that you've exceeded the [plan quota](https://docs.microsoft.com/en-us/visualstudio/online/resources/faq#what-are-visual-studio-codespacess-quotas). Deleting other existing plans in the subscription would allow you to create a new one.

> Visual Studio Codespaces is still showing no existing plans in the dropdown, so that the Create Codespace button just takes me to the menu to create a plan again.

The Codespaces portal only shows your plans, whereas the Azure portal shows all plans. Today, plans are associated with an individual user's identity, and are only usable by that identity (user). Is this a shared subscription by chance? Or did you log in with different credentials in the Codespaces service when creating the earlier plans? In any case, creating a new plan with the current identity should cause it to show up in the Codespaces portal.

We are working on the ability to share plans across multiple users (tracked in #146), so the view from the two portals will be more consistent and normal RBAC will apply for Codespaces plans.

**Comment by danysdragons** [comment]
Good morning @jkeech

I hadn’t realized I had created all those plans until I looked in the Azure portal - I had thought the process was just failing completely, so had come back to try again a few different times. I wasn’t using shared plans, and I have _never successfully created a Codepace up to this point_.

Before creating my issue I had removed the extra (Resource group, plan) pairs, and tried creating a Codepace again unsuccessfully. 

My experience closely parallels that of the gentleman WaldemarCoding from issue #452. I’ll link directly to the most relevant section of the discussion between him and yourself:

https://github.com/MicrosoftDocs/vsonline/issues/452#issuecomment-623154807

The screen animation he includes is the same as what I’m seeing at this point.

**Comment by danysdragons** [comment]
In issue https://github.com/MicrosoftDocs/vsonline/issues/452 @cmbrose mentioned a bug in the user lookup, and did an ad hoc update to unblock the user. Could something similar be done in my case?

**Comment by jkeech** [comment]
@cmbrose is going to look at the backend today and see if it's the same issue. If so, then yes we can fix it up for you. We're still working on the longer term fix that will fix up the small set of accounts that are in this state.

**Comment by cmbrose** [comment]
Yeah, it does look like the same error as before.

I've just don't the adhoc fix - @danysdragons can you check the Codespaces portal again now and check if you can now see your plan in the dropdown?  It should just appear, you shouldn't need to create a new one.

**Comment by danysdragons** [comment]
I do see the plans showing up in the dropdown now, and I was able to create a Codespace successfully just now. Thanks guys for taking care of this so quickly!

**Comment by cmbrose** [comment]
Excellent!  I'll go ahead and close this one up then
